doc/develop/extending.rst

```diff
@@ -313,7 +278,7 @@ scripts, or via a command-line interface.
 #. Install using ``pip install libtiff-0.4.2-cp27-cp27m-win_amd64.whl`` (or ``-win32``),
    downloaded from http://www.lfd.uci.edu/~gohlke/pythonlibs/#pylibtiff
 
-#.  Install OpenCV using ``pip install opencv_python-2.4.13.5-cp27-cp27m-win_amd64.whl`` (or ``-win32``),
+#.  Install OpenCV using ``conda install opencv -c conda-forge`` (or ``-win32``),
```

Comment:
What does the (or `-win32`) mean here? Should you change it to ``conda install opencv -c conda-forge -win32`` or ``conda install opencv -win32``

Reply:
`-win32` is just some meaningless left-over.